usr/tgtd.c

```diff
@@ -1085,6 +1085,38 @@ static int target_delete(const _ha_request *reqp, _ha_response *resp, void *user
 	return HA_CALLBACK_CONTINUE;
 }
 
+static int set_wan_latency(const _ha_request *reqp,
+	_ha_response *resp, void *userp)
+{
+	int rc = 0;
+	const char *latency_str;
+	uint32_t latency=20000; //default to 20 milliseconds
+
+	if (disallow_rest_call()) {
+		set_err_msg(resp, TGT_ERR_HA_MAX_LIMIT,
+		"Too many pending requests at TGT. Retry after some time");
+		return HA_CALLBACK_CONTINUE;
+	}
+	latency_str = ha_parameter_get(reqp, "latency");
```

Comment:
This must be fetched before disallow_rest_call() else if parameter fails it will return without decrementing active rest call count.